frameworks/PHP/one/composer.json

```diff
@@ -17,7 +17,7 @@
   ],
   "require": {
     "php": ">=7.1.0",
-    "lizhichao/one": ">=2.0.1",
+    "lizhichao/one": "dev-master",
```

Comment:
Please create a new version, and use it here.

Reply:
done